source/_patterns/00-atoms/block/heading/heading.twig

```diff
@@ -0,0 +1,5 @@
+<h{{ level|default(1) }} {%- include 'atoms-attributes' %}>
```

Comment:
Looked into using a macro for outputting HTML attributes, but Pattern Lab does the magic of [including the file](https://twig.symfony.com/doc/2.x/tags/macro.html#import) so we'll end up with problems using it outside.